.gitignore

```diff
@@ -0,0 +1,3 @@
+out/
+.idea/
```

Comment:
Nice. You didn't just ignore the `out` directory, but other useless stuff too. Good @mnaznin :) 👍